desc/experimental/_terpsichore.py

```diff
@@ -707,7 +707,7 @@ class TERPSICHORE(ExternalObjective):
         plasma. If TERPSICHORE fails to run, try decreasing this parameter. Default = 2.
     deltaJp : float, optional
         Resonance detuning parameter to resolve parallel current density singularities.
-        Default = 0.5.
```

Comment:
is there any intuition to be gained from understanding this default? like why did you lower it?

Reply:
TERPSICHORE documentation recommends a value in the range 1E-4 to 4E-2, I just accidentally had the default way too high. 

The inputs `awall` and `deltaJp` both artificially stabilize the plasma but make the numerics better, so typically you need to do a convergence scan in both parameters and the "best" value is problem dependent.